core/common/src/tile/TileMetadata.ts

```diff
@@ -70,7 +70,7 @@ export const defaultTileOptions: TileOptions = Object.freeze({
   enableInstancing: true,
   enableImprovedElision: true,
   ignoreAreaPatterns: false,
-  enableExternalTextures: false,
+  enableExternalTextures: true,

```

Comment:
We need to talk to @wgoehrig or @paulius-valiunas to ensure that tile regeneration agent will work properly when default tile tree flags are changed.

Reply:
I don't see why it would break. We use `defaultTileOptions` and only change `maximumMajorTileFormatVersion` before passing it to `iModelTileTreeIdToString` to get the tiles from blob storage, so any changes here should be reflected in the tileTrees we generate.

Reply:
> We use defaultTileOptions and only change maximumMajorTileFormatVersion before passing it to iModelTileTreeIdToString to get the tiles from blob storage

So when we change `defaultTileOptions` you will fail to find the old tiles?

Reply:
yeah that's a good point. If the change results in `iModelTileTreeIdToString` returning a different string, we would be looking at the wrong tiles in older changesets. How does the frontend handle this? Or are you supposed to bump the tile version when you change these defaults?

Reply:
Frontend just asks for tiles by Id computed from the options it is configured to use. Apps can freely change those options; tile format version is only relevant when we add new options.
Only the tile regeneration agent needs to know when default tile options change, so it can find old tiles using old options and produce new tiles using new options.
Just to be clear: the "external textures" flag is part of the **tile** Id, not the **tile tree** Id.

Reply:
Ok, so in this case it won't matter because we don't use it to generate tile ids, we only use it to get the tile tree id and then get the tile ids from blob storage.

In other cases that affect the tile tree id, I don't think there's anything we can do there, either. If every frontend can have different options configured, we don't have a way to retrieve those options so all we can do is use the defaults and hope that most frontends will use them too.

Reply:
yeah we should definitely improve it, I just never thought about the possibility of it changing the tile tree id. I think I may have an idea how to fix it, but that's outside the scope of this PR so let's take it offline. I'll start an email thread.